### PR TITLE
Update Spring Security Java API for new structure

### PR DESCRIPTION
**Pull request by alexisluque**

### CODE REVIEW

File: articles/quickstart/backend/java-spring-security/01-authorization.md [hunk 1]
```diff
@@ -91,20 +87,15 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
 ## Configure the Protected Endpoints
 
-The example below shows how to implement CRUD (create, read, update and delete) methods and a login route for the Photos resource. 
-
-For CRUD methods, you need authentication and specific scopes. 
+The example below shows how to implement API methods.
```
Comment (lbalmaceda) [inline articles/quickstart/backend/java-spring-security/01-authorization.md]: `implement secure API methods` or `secure your API methods`.

File: articles/quickstart/backend/java-spring-security/01-authorization.md [hunk 2]
```diff
@@ -91,20 +87,15 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
...
-The routes shown below are available for the following requests: 
+The routes shown below are available for the following requests:
 
-- `GET /login`: available for non-authenticated requests
-- `GET /photos`: available for authenticated requests containing an access token with the `read:photos` scope granted
-- `POST /photos`: available for authenticated requests containing an access token with the `create:photos` scope granted
-- `PUT /photos`: available for authenticated requests containing an access token with the `update:photos` scope granted
-- `DELETE /photos`: available for authenticated requests containing an access token with the `delete:photos` scope granted
-- Any other route that doesn't match the above requires the user to be authenticated with no additional scopes
+- `GET /api/public`: available for non-authenticated requests
```
Comment (lbalmaceda) [inline articles/quickstart/backend/java-spring-security/01-authorization.md]: This lines are not platform specific and are shared across multiple API quickstarts. Please move them to a snippet file

File: articles/quickstart/backend/java-spring-security/01-authorization.md [hunk 3]
```diff
@@ -152,97 +136,37 @@ import org.springframework.stereotype.Controller;
...
+        return new JSONObject()
+                .put("message", "All good. You can see this because you are Authenticated with a Token granted the 'read:messages' scope")
+                .toString();
     }
 
 }
 ```
 
 To build and run the seed project, use the command: `mvn spring-boot:run`.
-
```
Comment (lbalmaceda) [inline articles/quickstart/backend/java-spring-security/01-authorization.md:200]: Where is all this info that it's being removed going to be located now?

File: articles/quickstart/backend/java-spring-security/02-using.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+title: Using your API
```
Comment (lbalmaceda) [inline articles/quickstart/backend/java-spring-security/02-using.md:2]: I'd rather call this "calling your api". Same for the references below.

File: articles/quickstart/backend/java-spring-security/index.yml [hunk 5]
```diff
@@ -22,3 +22,4 @@ snippets:
   SecurePingController: server-apis/java-spring-security/SecurePingController
 articles:
   - 01-authorization
+  - 02-using
```
Comment (lbalmaceda) [inline articles/quickstart/backend/java-spring-security/index.yml:25]: See comment above.

File: articles/quickstart/backend/java-spring-security/02-using.md (hunk 4 above)
Comment (alexisluque) [reply articles/quickstart/backend/java-spring-security/02-using.md:2]: If I change the title, I'd also should change it for all Backend/API quickstarts to be consistent.

File: articles/quickstart/backend/java-spring-security/index.yml (hunk 5 above)
Comment (alexisluque) [reply articles/quickstart/backend/java-spring-security/index.yml:25]: Same comment as above.

File: articles/quickstart/backend/java-spring-security/01-authorization.md (hunk 3 above)
Comment (alexisluque) [reply articles/quickstart/backend/java-spring-security/01-authorization.md:200]: The info about using the API will be in `02-using.md`

**Review by aaguiarz — CHANGES_REQUESTED** [review]
The endpoints are not returning the text content we agreed, please fix that, thanks

**Review by rafanog — CHANGES_REQUESTED** [review]
Response messages do differ from the standard defined for this kind of qs

**Review by rafanog — APPROVED** [review]

**Pull request merged**